tools/explorer/test/run_tests.py

```diff
@@ -13,11 +13,19 @@
 PORT = 8002
 COMMAND_URL = "http://" + HOST + ":" + str(PORT) + "/apipost/v1/send_command"
 TEST_LOAD_MODEL_PATHS = [
-    "test/ttmlir/Dialect/TTNN/mnist_sharding.mlir",
-    "tools/explorer/test/models/*.mlir",
+    "test/ttmlir/Dialect/TTNN/optimizer/mnist_sharding.mlir",
```

Comment:
I don't think we need ones from optimizer dir anymore? Should probably keep them separate going forward.